include/graph/graph.hpp

```diff
@@ -18,6 +18,11 @@ class LayerExample {
   LayerExample(int id1, int type1) : id_(id1), type_(type1) {}
   int checkID() const { return id_; }
   void In(const std::vector<int>& a) { primer_ = a; }
+  void Work() {
+    for (int& i : primer_) {
+      i += 1;
+    }
+  }
```

Comment:
Please, do not add anything which is needed only for specific algorithm directly to the graph structure

Reply:
And then how do I write a test to check, or is it not necessary here? I added this to the example of a layer that will be deleted later when the real written layers are connected.

Reply:
You can do whatever you want with the graph, right? But here you say that the whole library can work only this way. You must not specify the workload inside the library. User should be able to modify it without modifying the source code

Reply:
done